integration-cli/docker_cli_import_test.go

```diff
@@ -124,3 +124,25 @@ func (s *DockerSuite) TestImportFileNonExistentFile(c *check.C) {
 	_, _, err := dockerCmdWithError("import", "example.com/myImage.tar")
 	c.Assert(err, checker.NotNil, check.Commentf("import non-existing file must failed"))
 }
+
+func (s *DockerSuite) TestImportWithQuotedChanges(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	dockerCmd(c, "run", "--name", "test-import", "busybox", "true")
+
+	temporaryFile, err := ioutil.TempFile("", "exportImportTest")
+	c.Assert(err, checker.IsNil, check.Commentf("failed to create temporary file"))
+	defer os.Remove(temporaryFile.Name())
+
+	runCmd := exec.Command(dockerBinary, "export", "test-import")
+	runCmd.Stdout = bufio.NewWriter(temporaryFile)
+
+	_, err = runCommand(runCmd)
+	c.Assert(err, checker.IsNil, check.Commentf("failed to export a container"))
```

Comment:
You can this with `pkg/integration/cmd` which will gives you a better error message when it fails:

```go

result := icmd.RunCmd(icmd.Cmd{
    Command: binaryWithArgs("export", "test-import"),
    Stdout: bufio.NewWriter(temporaryFile),
})
c.Assert(result, icmd.Matches, icmd.Success)

```